packages/ui/src/views/repo/pull-request/details/components/conversation/pull-request-timeline-item.tsx

```diff
@@ -6,13 +6,15 @@ import { getInitials } from '@utils/utils'
 
 import { PullRequestCommentBox } from './pull-request-comment-box'
 
-interface TimelineItemProps {
-  header: {
-    avatar?: ReactNode
-    name?: string
-    description?: ReactNode
-    selectStatus?: ReactNode
-  }[]
+export interface TimelineItemPropsHeaderType {
+  avatar?: ReactNode
+  name?: string
+  description?: ReactNode
+  selectStatus?: ReactNode
+}
+
+export interface TimelineItemProps {
```

Comment:
Probably no need to export `TimelineItemPropsHeaderType` since you can get it from `TimelineItemProps` using `TimelineItemProps['header']`